src/analysis/locate.ml

```diff
@@ -1016,7 +1016,16 @@ let get_doc ~config ~env ~local_defs ~comments ~pos =
         Format.fprintf fmt "]\n"
       );
     begin match
-      Ocamldoc.associate_comment comments loc !last_location
+      let (_, deepest_before) = List.hd @@ Mbrowse.deepest_before loc.loc_start [browse] in
```

Comment:
Nitpick: that's a lot going on in the `match ... with` argument, could you give it a name and then match on this ?
Or just stay closer to  the current code with something like:
```ocaml
let after_only = 
  .....
in 
begin match
  Ocamldoc.associate_comment ~after_only comments loc !last_location
with